code/nxsOnto/nxsOnto/generator.py

```diff
@@ -184,12 +175,12 @@ def addFieldToDict(classDict, field, defn_name): # make a function to be reused
         #print('~~~ field did not exist: %s' % long_name)
         classDict[className]['fields'][long_name] = {} # create dictionary for field if doesn't exist
```

Comment:
Nicer to get the sub-dictionary `classDict[className]['fields']` once rather than using the same copy and pasted expression. This applies for all dictionary access below too.